### ISSUE TITLE
Isorespin script download from google drive hangs

### ISSUE THREAD

**Issue by Xunius** [stockmind/gpd-pocket-ubuntu-respin#82]
Hi, 
Thanks for creating such a cool tool.
I'm following the instructions till the `./build.sh` step, which hanged at 
```
Isorespin script not found. Downloading it...
--2017-11-21 19:37:08--  https://drive.google.com/uc?export=download&id=0B99O3A0dDe67S053UE8zN3NwM2c
Loaded CA certificate '/etc/ssl/certs/ca-certificates.crt'
Resolving drive.google.com... 46.82.174.68
Connecting to drive.google.com|46.82.174.68|:443... 
```

Then I get `Connection timed out` errors.
It might be my network issue, but I wonder if you could move that script into this repo if it is not too big?

**Comment by Xunius** [comment]
Yes it was my network problem. I managed to get it downloaded in office.

**Comment by stockmind** [comment]
Not sure on how to handle this! As if there are network issues that prevent isorespin to download, you will also have network issues downloading all the packages that will be installed on system during respin

**Comment by Xunius** [comment]
Yes. I guess that's due to I'm accessing the internet from behind a "great" firewall, all google-relatead things are cut off, github is fine, many other linux things are fine, vpn is not always stable, so. But it is installed and runs very nicely now.
